packages/react-composites/src/composites/ChatComposite/selectors/baseSelectors.ts

```diff
@@ -2,8 +2,34 @@
 // Licensed under the MIT license.
 import { CommunicationIdentifierKind } from '@azure/communication-common';
 import { ChatAdapterState } from '../adapter/ChatAdapter';
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+import { FileUploadState } from '../file-sharing';
 
 /**
  * @private
  */
 export const getUserId = (state: ChatAdapterState): CommunicationIdentifierKind => state.userId;
+
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+/**
+ * @private
+ */
+export const getFileUploads = (state: ChatAdapterState): FileUploadState[] | undefined => {
```

Comment:
These selectors use the `state` directly. This means they'll run on *every render*. But file upload state is unlikely to change that frequently.

I'd recommend that you add a selector to get just the file upload state `state?.fileUploads` and have these depend on that selector (so they won't run most of the time).